config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customers#create"
```

Comment:
As I mentioned in the other comments, this would be a more RESTful option for adding an item to a customer:
`post "/customers/:customer_id/items, to: "customer_items#create"`
Note the /items on the path and the new joins controller it's routed to